infra/Pos/DHT/Real/Real.hs

```diff
@@ -97,9 +97,9 @@ startDHTInstance KademliaDHTInstanceConfig {..} = do
 
     log' logF =  usingLoggerName ("kademlia" <> "messager") . logF . toText
     createKademlia port key cfg =
-        K.createL (fromIntegral port) key cfg (log' logDebug) (log' logError)
+        K.createL "127.0.0.1" (fromIntegral port) key cfg (log' logDebug) (log' logError)
     createKademliaFromSnapshot port cfg snapshot =
-        K.createLFromSnapshot (fromIntegral port)
+        K.createLFromSnapshot "127.0.0.1" (fromIntegral port)
```

Comment:
Both of these ought to be `0.0.0.0` (that's what the Kademlia library chose before).